perf/impl/js-libp2p/v0.45/perf

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Find the path to the Node.js executable
+node_path=$(which node)
+
+# Check if Node.js is installed
+if [ -z "$node_path" ]; then
+  echo "Node.js is not installed or not in the PATH."
+  # Update package list and install Node.js
+  sudo yum update -y
+  sudo yum install -y gcc-c++ make
+  curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
+  sudo yum install -y nodejs
+
+fi
```

Comment:
Seems like installing NodeJS is failing:

https://github.com/libp2p/test-plans/actions/runs/5868079634/job/15910134917

@maschad are you seeing the same issue when running against your AWS instances?

Reply:
This [error](https://github.com/libp2p/test-plans/actions/runs/5868079634/job/15910134917#step:12:4296) is actually when parsing the output of the tests, not related to installing node on the AWS.

I don't get this error when running with my AWS infrastructure so this is strange as I was able to generate the [benchmark results](https://github.com/libp2p/test-plans/pull/244/files#diff-c5b04dce18f8d7b44769534ed718164efaf58396d8f9031af22169b5421f646b)

Reply:
So I was able to reproduce this issue on my instance, it seems as if the runner script is executed before the Node js installation is complete. I wonder if this is the best place to install node or if we should add it to the [terraform template](https://github.com/libp2p/test-plans/blob/perf/js-libp2p/perf/terraform/configs/local/terraform.tf#L16-L30) like:

```
user_data = <<-EOF
              #!/bin/bash
              sudo yum update -y
              sudo yum install -y git nodejs npm
              EOF
```

Reply:
Both the server and the client need to install NodeJS before they can execute the server or client software respectively. Maybe the client is done installing NodeJS faster than the server, then tries to connect to the server, but given that the server is still installing NodeJS, it is not yet ready to receive the request. Could that be the case @maschad?

We could install `nodejs` inside of [`user-data.sh`](https://github.com/libp2p/test-plans/blob/master/perf/terraform/modules/long_lived/files/user-data.sh).

That said, `user-data.sh` is part of the long-lived infrastructure. Thus, when one e.g. wants to change the NodeJS version, it is not as simple as editing `user-data.sh`. Instead one would need to run `terraform apply` in `config/remote` before the change takes effect.

How about installing NodeJS in the `impl/js-libp2p/v0.45` `Makefile` as part of the installation (i.e. `make`) instead. Maybe a hack, but gets the job done. At least for now.

Reply:
Good idea @mxinden I've made that change to the `Makefile` in 0a220bfabc2cd13e93aa298d521c31106df02128